pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -127,6 +127,47 @@ spec:
             - name: socket-dir
               mountPath: /csi
         {{ end }}
+        {{ if .EnableCSIAddonsSideCar }}
+        - name: csi-addons
+          image: {{ .CSIAddonsImage }}
+          args :
+            - "--node-id=$(NODE_ID)"
+            - "--v=5"
+            - "--csi-addons-address=$(CSI_ADDONS_ENDPOINT)"
+            - "--controller-ip=$(POD_IP)"
+            - "--controller-port={{ .CSIAddonsMetricsPort }}"
```

Comment:
the variable `CSIAddonsMetricsPort` is misnamed, just `CSIAddonsPort` would be more appropriate